docs/reference/logging/index.md

```diff
@@ -25,6 +25,20 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
+
+    --log-opt max-size=[0-9+][k|m|g]
+    --log-opt max-file=[0-9+]
+
+Logs that reach `max-size` are rolled over. You can set the size in kilobytes(k), megabytes(m), or gigabytes(g). eg `--log-opt max-size=50m`. If `max-size` is not set, then logs are not rolled over.
+
+
+`max-file` specifies the maximum number of files that a log is rolled over before being discarded. eg `--log-opt max-file=100`. If `max-size` is not set, then `max-file` is not honored.
+
+If `max-size` and `max-file` are set, `docker logs` only returns the log lines from the newest log file. 
```

Comment:
I think this information really belongs in `logs.md`. In fact, I would like `logs.md` to provide some description of rollover feature and how that impacts the behavior of options. For example, `--tail`` description says that when an invalid value is passed, the value is set to `latest`. However, you can't tell what `latest` means just by looking at the man page.